services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/RouteServiceImpl.java

```diff
@@ -0,0 +1,8 @@
+package com.futurewei.alcor.route.service.Impl;
+
+import com.futurewei.alcor.route.service.RouteService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RouteServiceImpl implements RouteService {
```

Comment:
empty implementation?

Reply:
Yes, because we haven't any route logic service yet